src/robot_interface/telemetry/mqtt_client.py

```diff
@@ -74,7 +75,7 @@ def run(self, isar_id: str, robot_name: str) -> None:
             try:
                 payload = self.telemetry_method(isar_id=isar_id, robot_name=robot_name)
                 topic = self.topic
-            except RobotTelemetryPoseException:
+            except (RobotTelemetryPoseException, RobotTelemetryNoUpdateException):
```

Comment:
```suggestion
            except (RobotTelemetryException):
```

Reply:
The RobotTelemetryException is a more general exception that is caught below and sends a cloud health topic to flotilla